core/src/main/java/jenkins/management/CliLink.java

```diff
@@ -52,4 +56,9 @@ public String getDescription() {
     public String getUrlName() {
         return "cli";
     }
+
+    @Override
+    public Permission getRequiredPermission() {
+        return Jenkins.MANAGE;
+    }
```

Comment:
this always struck me as odd.  the CLI is available for everyone to use and each CLI should have its own authz on who can use it inside the implementation - but users have no way to actually discover these. 

this is not related to this PR - it has not made this worse, just a note.

Reply:
Right, basically just limiting the discovery of the CLI page. That said, this change is correctly not hiding the link from Manage-rs.